docs/deployment_and_config.md

```diff
@@ -262,6 +298,38 @@ Replace the entries in `<>` with your values:
   and warnings about expired certs on Let's Encrypt.
 - `nginx_defualt_fqdn` (optional): enable a default host (404 page not found).
 
+Slack notification channels configuration go to [OpenSearch documentation](https://docs.opensearch.org/docs/latest/observing-your-data/notifications/api/) for more info.
+
+- `channels`: Channel configurations
```

Comment:
If this is related to opensearch, these variables should have that prefix, right? Maybe also use `notification_channels` instead of `channels`.